components/trustpilot/sources/new-product-reviews/new-product-reviews.mjs

```diff
@@ -1,40 +1,103 @@
-import {
```

Comment:
Why are all these source components being removed? Are they not working as expected?